src/constants/WalletAndCurrencyConstants.js

```diff
@@ -282,6 +282,16 @@ export const SPECIAL_CURRENCY_INFO: {
     noMaxSpend: true,
     isImportKeySupported: false
   },
+  mumbai: {
```

Comment:
Optional: We are missing `dummyPublicAddress` and `walletConnectV2ChainId`, compared to what kovan has.